style.css

```diff
@@ -186,30 +186,73 @@ hr {
 	right: 10px;
 }
 
-/*-------------Login Content Box----------------*/
+/*-------------Content Boxes----------------*/
+
+.main_content{
+	grid-column: 1 / span 6;
+	grid-row: 2;
+	background: #3299a8;
+	display: grid;
+	grid-template-rows: 13vw 20vw;
+	grid-template-columns: 14.5vw 20vw 5vw 20vw 5vw 20vw 5vw;
+}
+
+.sub_content{
+	grid-column: 1 / span 3;
+	grid-row: 1;
+	background: blue;
+	display: grid;
+	grid-template-rows: 13vw 20vw;
+	grid-template-columns: 14.5vw 20vw 5vw 20vw 5vw 20vw 5vw;
+	z-index: 2;
+}
+
+.title1{
+	background-color:;
+	padding-top: 0vw;
+	font-size: 4vw;
+	grid-column: 1 / span 8;
+	grid-row: 1;
+}
+
+.title2{
+	background-color: #34c747;
+	padding-left: 3vw;
+	padding-top: 2vw;
+	font-size: 4vw;
+	grid-column: 1 / span 8;
+	grid-row: 1;
+}
 
-.content_1{
+.content_2{
 	box-sizing: border-box;
-	margin: -2vw 5vw 0vw;
-	width: 80%;
-	height: 35vw;
 	background-color: #f7f7f7;
-	z-index: 2;
-	padding-left: 15px;
-	padding-right: 10px;
 	background: rgba(204, 204, 204, 0.7);
-	grid-column: 3/5;
+	grid-column: 2;
 	grid-row: 2;
 }
 
-.image_box1{
+.content_3{
```

Comment:
For content_3 and content_4 I have added new code and deleted what was not needed so the classes work better with the design.